indico/modules/categories/controllers/management.py

```diff
@@ -117,6 +122,33 @@ def _process(self):
                                                     icon_form=icon_form, logo_form=logo_form)
 
 
+class RHAPIEventMoveRequests(RHManageCategoryBase):
+    def _process_GET(self):
+        return EventMoveRequestSchema(many=True).jsonify(self.category.event_move_requests)
+
+    @use_kwargs({
+        'state': EnumField(MoveRequestState, required=True)
+    })
+    def _process_POST(self, state):
```

Comment:
Let's reurn something - event if it's just `return '', 204` for a *no content* response